### PR TITLE
Fix sed command and GitHub URL

### PR DESCRIPTION
**Pull request by mjgiarlo**
I found I needed these changes in order to 1. edit out the branch prompt in config/deploy.rb, and 2. to be able to pull down private repos without being asked for HTTPS credentials. It otherwise worked like a charm.

May I ask someone on @sul-dlss-labs/infrastructure-team who's successfully run sdr-deploy to try again w/ this branch?

### CODE REVIEW

File: deploy.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def repo_infos
     puts "Deploying #{repo_dir}"
     `bundle install`
     # Comment out where we ask what branch to deploy. We always deploy master.
-    `sed -i '' 's/^\\(ask :branch.*\\)/#\\1/g' config/deploy.rb`
+    `sed -i -e 's/^\\(ask :branch.*\\)/#\\1/g' config/deploy.rb`
```
Comment (jcoyne) [inline deploy.rb:81]: This will work for GNU sed, but doesn't work on osX (BSD sed)

Comment (mjgiarlo) [reply deploy.rb:81]: @jcoyne as I feared. I'll replace this with a Ruby incantation.

**Comment by justinlittman** [comment]
Worked for me.

File: deploy.rb (hunk 1 above)
Comment (mjgiarlo) [reply deploy.rb]: Change pushed. Have another look, @jcoyne?

**Pull request merged**